### DISCUSSION TITLE
关于动态编译和静态使用的一些问题

### DISCUSSION THREAD

**Discussion by wwwang97** [BYJRK/DotNet-Discussions#96] [C# 技术交流]
各位大佬们，请教一下，在程序中引入了某个DLL，然后我想再程序运行中去替换这个DLL，有办法做到么？

**Comment by dumbnessrf** [comment]
不是本来就可以吗，只要DLL同名，exe会自动去找

**Comment by Dynesshely** [comment]
请详细地指出 `替换` 操作是指 ?
请问是文件层面替换 dll 文件 ?
还是在 AppDomain 中替换程序集 ?
